data/json/recipes/weapon/ranged.json

```diff
@@ -17,7 +17,68 @@
     "components": [
       [ ["nailgun", 1] ],
       [ ["pipe", 1] ],
-      [ ["2x4", 2], [ "stick", 3 ] ]
+      [
+        ["2x4", 2],
+        [ "stick", 3 ]
+      ]
+    ]
+  },
+  {
+    "result": "carbine_flintlock_double",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "skills_required": [ "gun", 3 ],
+    "time": 30000,
+    "autolearn": true,
+    "book_learn": [[ "manual_rifle", 3 ] , [ "mag_rifle", 4 ]],
+    "qualities":[
+      { "id":"HAMMER","level":2 },
+      { "id":"SAW_M_FINE","level":1 },
+      { "id":"SCREW","level":1}
+    ],
+    "tools": [
+      [
```

Comment:
Welding or forging but not both. Preferably forging. Try `using: [ [ "forging_standard", 5 ] ]` instead. This means if we add more forges or change their efficiency your recipe won't be become outdated